operators/multiclusterobservability/api/shared/multiclusterobservability_shared.go

```diff
@@ -44,6 +44,14 @@ type ObservabilityAddonSpec struct {
 	// +kubebuilder:validation:Maximum=3600
 	Interval int32 `json:"interval,omitempty"`
 
+	// Workers is the number of workers that work parallelly to push metrics to hub server.
```

Comment:
nit: Workers is the number of workers that work in parallel ...   sounds more natural than the way this is written